primitives/src/script/mod.rs

```diff
@@ -36,9 +33,18 @@ pub const MAX_REDEEM_SCRIPT_SIZE: usize = 520;
 pub const MAX_WITNESS_SCRIPT_SIZE: usize = 10_000;
 
 hashes::hash_newtype! {
-    /// A hash of Bitcoin Script bytecode.
+    /// A 160-bit hash of Bitcoin Script bytecode.
+    ///
+    /// Note: there is another "script hash" object in bitcoin ecosystem (Electrum protocol) that
+    /// uses 256-bit hash and hashes a semantically different script. Thus, this type cannot
```

Comment:
```suggestion
    /// Note: there is another "script hash" object in the bitcoin ecosystem (Electrum protocol)
    /// that uses 256-bit hash and hashes a semantically different script. Thus, this type cannot
```

Adds 'the' in front of 'bitcoin eccosystem and maintains 100 column width.

Reply:
I suggest we do these later unless I have to push. This PR blocks script tagging and while I don't have it ready yet, I'd like to parallelize our work.

Reply:
Sweet as.